specification/eventgrid/data-plane/readme.md

```diff
@@ -144,6 +144,7 @@ input-file:
 - Microsoft.ResourceNotifications/stable/2018-01-01/HealthResources.json
 - Microsoft.ResourceNotifications/stable/2018-01-01/Resources.json
 - Microsoft.AVS/stable/2018-01-01/PrivateCloud.json
+- Microsoft.ApiCenter/stable/2024-03-01/ApiCenter.json
```

Comment:
Please add it for each tag section. Also need to use 2018-01-01 as the version.

Reply:
Changed.